roles/consul/tasks/nginx_proxy.yml

```diff
@@ -1,9 +1,20 @@
 ---
+- name: create nginx configuration directories
+  sudo: yes
+  file:
+    state: directory
+    path: "{{ item }}"
+  with_items:
+    - /etc/nginx
+    - /etc/nginx/templates
+  tags:
+    - consul
+
 - name: deploy nginx template
```

Comment:
Should probably do this to `marathon/nginx_proxy.yml` as well to keep things consistent.